content/javascript/concepts/strings/terms/to-string/to-string.md

```diff
@@ -0,0 +1,48 @@
+
+Title: '.toString()'
+Description: 'Returns a string representing the object'
+Subject: 
+ - 'Computer Science'
+Tags:
+ -  'Functions'
+ -  'Strings'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+
```

Comment:
Could we remove the empty line 13 or 14 so there is just one empty space between the ending dashes of the metadata and introduction?